src/udev/udev-node.c

```diff
@@ -514,8 +518,10 @@ static int link_update_diskseq(sd_device *dev, const char *slink, bool add) {
 }
 
 static int link_update(sd_device *dev, const char *slink, bool add) {
+        _cleanup_(rmdir_and_freep) char *path = NULL; /* #1 */
```

Comment:
"path" might be a bit generic, could be the dir or the lock file, maybe be explicit here, i.e. dirpath? or so?